docs/pages/index.html

```diff
@@ -40,6 +40,14 @@
     <a href="{{ site.baseurl }}/concepts">
         <div class="overview">Concepts</div>
     </a>
+    <ul><b>Concepts</b>
```

Comment:
This set of changes is from a different collection of commits. Ideally, this set of changes is not included in this PR.

Not a big deal in this case, so we're ok.